btr-web/btr-main-app/components/person-info/Details.vue

```diff
@@ -9,13 +9,18 @@
         {{ item.email }}
       </div>
     </div>
-    <div v-if="item.address" class="flex">
+    <div
```

Comment:
Are you sure we dont want to display other parts of the address if there is no country in the address ?

Reply:
The problem here is the icon will show if there is no information because address will be a populated object with keys like city: '' and then the icon shows with nothing and I didn't want to add all of the parts to the v-if.

Therefore I've added a computed value with all of the bits and am using that instead